clients/vscode/src/chat/ChatViewProvider.ts

```diff
@@ -18,20 +18,23 @@ import {
   commands,
   ColorThemeKind,
 } from "vscode";
-import type { ServerApi, ChatMessage, Context, NavigateOpts } from "tabby-chat-panel";
+import type { ServerApi, ChatMessage, Context, NavigateOpts, FocusKeybinding } from "tabby-chat-panel";
 import hashObject from "object-hash";
 import * as semver from "semver";
 import type { ServerInfo } from "tabby-agent";
 import type { AgentFeature as Agent } from "../lsp/AgentFeature";
 import { createClient } from "./chatPanel";
 import { GitProvider } from "../git/GitProvider";
-
+import { getLogger } from "../logger";
+import { contributes } from "../../package.json";
+import { parseKeybinding, readUserKeybindingsConfig } from "../util/KeybindingParser";
 export class ChatViewProvider implements WebviewViewProvider {
   webview?: WebviewView;
   client?: ServerApi;
   private pendingMessages: ChatMessage[] = [];
   private pendingRelevantContexts: Context[] = [];
   private isChatPageDisplayed = false;
+  private isUnix = Boolean(env.appHost === "desktop" && env.appRoot && env.appRoot[10] === "/");
```

Comment:
```suggestion
  // FIXME: this check is not compatible with the environment of a browser in macOS
  private isMac = Boolean(env.appHost === "desktop" && process.platform === "darwin");
```

I do not think this makes sense.   
Using `isMac = process.platform == 'darwin'` works on desktop Node.js, but not on a browser, as it always returns false on a browser, even on a browser in macOS.   
The updated implementation with `env.appRoot` also always returns false on a browser.  
So we need another way to check if it is macOS on browser, for example use the browser UserAgent.   

It looks good to me that not support the isMac check on browser for now. But please leave a fixme comment stating that this is not compatible with a browser environment yet.